tools/remove_chimera/remove_chimera.py

```diff
@@ -195,6 +202,9 @@ def write_summary( summary_file, results_chimera ):
     # Inputs
     group_input = parser.add_argument_group( 'Inputs' )
     group_input.add_argument( '-f', '--input-fasta', required=True, help='The cluster sequences (format: FASTA).' )
+    # group_exclusion_filter = group_input.add_mutually_exclusive_group()
```

Comment:
why is that commented ? If useless, why not removing it ?
the same for the next 2 lines